src/core/components/base-header.vue

```diff
@@ -14,6 +17,37 @@ const router = useRouter();
 const auth = useAuth();
 const user = ref(auth.user);
 
+const confirm = useConfirm();
+const toast = useToast();
+
+const handleConfirmation = () => {
+  confirm.require({
+    header: "Account verification",
+    message:
+      "Do you want your account to be verified by WAW? These action might take weeks and it's irreversible.",
+    icon: PrimeIcons.EXCLAMATION_CIRCLE,
+    accept: () => {
+      toast.add({
+        severity: "success",
+        summary: "Verification requested",
+        detail:
+          "You are now under verification proccess. We'll get back to you by email soon!",
+        life: 3000,
+      });
+    },
+    reject: () => {
+      toast.add({
+        severity: "warn",
```

Comment:
I think this just should be an info message, a warning is a little bit too much.

Reply:
Alright, got it!